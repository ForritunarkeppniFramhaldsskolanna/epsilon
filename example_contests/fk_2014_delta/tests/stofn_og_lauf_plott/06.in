5
53 47 49 46 42
