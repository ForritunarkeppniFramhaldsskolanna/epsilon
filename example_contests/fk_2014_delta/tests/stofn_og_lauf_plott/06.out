4 2679
5 3
